shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -206,11 +206,12 @@ - (CGRect)accessibilityFrame {
 #pragma mark - UIAccessibilityElement protocol
 
 - (id)accessibilityContainer {
-  if ([self hasChildren]) {
+  if ([self hasChildren] || _uid == kRootNodeId) {
     if (_container == nil)
       _container = [[SemanticsObjectContainer alloc] initWithSemanticsObject:self bridge:_bridge];
     return _container;
   }
+  NSAssert(_parent != nil, @"illegal access to parent");
```

Comment:
Given that the only case we expect the parent to be nil would be the root node, maybe "Illegal access to root semantics node parent"?